### ISSUE TITLE
Autoload keymap display problem with echo-keystrokes and which-key

### ISSUE THREAD

**Issue by noctuid** [noctuid/general.el#29]
The first time the key bound to the "autoloaded" keymap is run, it works correctly, but the wrong which-key menu may be displayed. For example, if an "autoload" for `projectile-command-map` is bound to `SPC p`, then only `SPC-` will be displayed in the echo area and the keys under `SPC` will be shown in which-key. This problem was discovered in #26.

**Comment by blaenk** [comment]
See [this comment](https://github.com/noctuid/general.el/issues/26#issuecomment-226009975). I figured out a fix! It's up to you to include it now :dancer:

**Comment by noctuid** [comment]
From the emacs manual:

> An element of the form (t . event) forces event to be added to the current command’s key sequence. 

I wasn't actually sure what this meant before, but thanks for figuring out a fix!

> Can you rephrase or elaborate?

What I meant is that which-key would also act as if only `SPC` and not `SPC p` had been pressed.
